### DISCUSSION TITLE
Rigidbody + Concave Collision hull doesn't collide with Static Concave mesh?

### DISCUSSION THREAD

**Discussion by JaineshPathak** [DanielChappuis/reactphysics3d#321] [Q&A]
Hello.

I have a chair mesh and another collision hull mesh which is low-poly version of original chair mesh. 
![image](https://user-images.githubusercontent.com/70505458/222717233-cd522011-261c-453f-b3a5-d40437cdbf32.png)

I am facing problem rigidbody with concave collision shape doesn't collide with static collision body (Eg: level) with concave collision shape. It simply ignores collision.

I also double-checked the collision category and mask bits. They are all fine.
However, dynamic concave objects collide fine with basic shapes like Box, Sphere, Capsules, etc.

Thank you.

**Comment by DanielChappuis** [comment]
Yes, a ConcaveMeshShape is not able to collide against another ConcaveMeshShape. This is because this kind of collision is really expensive in general.

A ConcaveMeshShape should only be used as a static mesh to describe a level for instance. The dynamic bodies should only have colliders with convex shapes (BoxShape, SphereShape, CapsuleShape, ConvexMeshShape).
